packages/console/src/VncConsole/VncConsole.test.js

```diff
@@ -4,6 +4,6 @@ import { shallow } from 'enzyme';
 import VncConsole from './VncConsole';
 
 test('placeholder render test', () => {
-  const view = shallow(<VncConsole />);
+  const view = shallow(<VncConsole host="my.unknown.host" />);
   expect(view).toMatchSnapshot();
```

Comment:
Can we add tests to modify some of the props and verify the expected results? Simulate the connecting, disconnected, and connected states, etc.